paparazzi/paparazzi/src/main/java/app/cash/paparazzi/internal/PaparazziCallback.kt

```diff
@@ -74,7 +74,7 @@ internal class PaparazziCallback(
             } else if (type.isArray && type.componentType == Int::class.javaPrimitiveType) {
               // Ignore.
             } else {
-              logger.error(null, "Unknown field type in R class: $type")
+              logger.error(null, "Unknown field type ($type) in R class: $field")
```

Comment:
Fair point, then l suggest updating the message text like so:
```suggestion
              logger.error(null, "Unknown type ($type) in R class field: $field")
```